presto-hive-hadoop2/pom.xml

```diff
@@ -119,7 +124,26 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludedGroups>hive-s3</excludedGroups>
+                            <groups>hive</groups>
```

Comment:
previously it was running all but s3 tests, now it is running only `hive` tests.

To me it is ok. I do not know what other think about it.

Reply:
There are just two test classes here, `hive` and `hive-s3`